### PR TITLE
Add functionality to save vaccine card image AB#11281

### PR DESCRIPTION
**Pull request by MikeLyttle**
# Implements [AB#11281](https://dev.azure.com/qslvic/304a1f8c-dace-4f85-adf3-bf563d5b3a39/_workitems/edit/11281)

-   [ ] Enhancement
-   [ ] Bug
-   [ ] Documentation

## Description

Adds saving images of vaccine cards via the html2canvas library and adjusts spacing on the vaccine card.

If you are reviewing this PR, please adhere to the guidelines as [documented](https://github.com/bcgov/healthgateway/wiki/prguidance).

## Testing

-   [ ] Unit Tests Updated
-   [ ] Functional Tests Updated
-   [ ] Not Required

### Steps to Reproduce

If this is a bug, please provide details on how to reproduce the code.

### UI Changes

YES

### Browsers Tested

-   [x] Chrome
-   [ ] Safari
-   [ ] Edge
-   [ ] Firefox

Provide an explanation for any test(s) not completed.

## Notes/Additional Comments

Any additional notes or comments that may be relevant. Include any specialized deployment steps here.

## Items to Review:

-   [General PR Guidelines](https://github.com/bcgov/healthgateway/wiki/PRguidance)

-   Fulfills Work Item Requirements

    -   The changes meet/implement story's acceptance criteria. Fixes identified problems if bug.
    -   Changes not directly related to the task requirements need to be documented on the PR.

-   Compilation / Tests

    -   The changes do not break compilation and/or unit or functional tests; unless other item addresses them specifically.

-   Logic Problems / Functional Behaviour

    -   The changes work as intended and do not introduce problems.

-   Performance

    -   The changes do not introduce obvious performance issues.

-   Documented Standards

    -   The changes adhere to the team's documented [Coding Standards](https://github.com/bcgov/healthgateway/wiki/standards).

-   Readability / Maintainability
    -   The changes can be easily understood/read and allow for future enhancements without major refactoring. Readability is preferred over "clever code".
    -   Reasoning for changes needs to be clearly articulated. Disagreements should be arbitrated by a third developer.

### CODE REVIEW

**Review by sslaws — APPROVED** [review]

File: Apps/WebClient/src/ClientApp/src/components/modal/vaccineCardModal.vue [hunk 1]
```diff
@@ -175,37 +164,35 @@ export default class VaccineCardModalComponent extends Vue {
...
-                            saveAs(blob, "BCVaccineRecord.pdf");
+                            saveAs(blob, "BCVaccineCard.png");
                         });
                     });
-                }
-            })
-            .finally(() => {
-                this.isDownloading = false;
-            });
+                })
```
Comment (BrianMaki) [inline Apps/WebClient/src/ClientApp/src/components/modal/vaccineCardModal.vue:191]: Was the indentation intentional for the new code?  New code added does not seem to have the same column alignment as with the previously deleted code.

File: Apps/WebClient/src/ClientApp/src/views/publicVaccineCard.vue [hunk 2]
```diff
@@ -156,36 +149,34 @@ export default class PublicVaccineCardView extends Vue {
...
                     });
-                }
-            })
-            .catch((err) => {
-                this.logger.error(`Error retrieving vaccine card PDF: ${err}`);
-            })
-            .finally(() => {
-                this.isDownloading = false;
-            });
+                })
```
Comment (BrianMaki) [inline Apps/WebClient/src/ClientApp/src/views/publicVaccineCard.vue:175]: See previous comment about alignment - vaccineCardModal.vue.

**Review by BrianMaki — APPROVED** [review]
Some minor comments about alignment.

File: Apps/WebClient/src/ClientApp/src/components/modal/vaccineCardModal.vue (hunk 1 above)
Comment (MikeLyttle) [reply Apps/WebClient/src/ClientApp/src/components/modal/vaccineCardModal.vue:191]: Formatting and indentation is handled by prettier in the front-end.  I use the prettier extension in VS Code, so the files are auto-formatted upon save.

**Pull request merged**